modules/orchestrator/proc-deploying-workflows-on-a-cluster.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-deploying-workflows-on-a-cluster.adoc_{context}"]
+= Deploying workflows on a cluster
+
+You can now deploy the workflow on a cluster, since the image has been pushed to the image registry and the deployment manifests have been made available.
+
+.Prerequisites
+
+* You have an {ocp-short} cluster with the following versions of components installed:
+
+*   {product} ({product-very-short}) `v1.5`
+*   Orchestrator plugins `v1.4` or `v1.5`
+*   OpenShift Serverless `v1.35`
+*   OpenShift Serverless Logic `v1.35`
+
+For further instructions on installing these components, see the {orchestrator-book-link}#con-orchestrator-plugin-components[Orchestrator plugin components on {ocp-short}].
+
+* You must apply the workflow manifests in a namespace that contains a `SonataflowPlatform` custom resource (CR), which manages the link:https://sonataflow.org/serverlessworkflow/latest/cloud/operator/supporting-services.html[supporting services]. By default, OpenShift Serverless Logic services are installed in the `sonataflow-infra` namespace.
```

Comment:
What do you think about using https://docs.redhat.com/en/documentation/red_hat_openshift_serverless/1.36/html-single/serverless_logic/index#serverless-logic-configuring-workflow-services